frontend/components/Domain/Recipe/RecipeIngredientListItem.vue

```diff
@@ -45,6 +45,23 @@ export default defineComponent({
   .d-inline {
     & > p {
       display: inline;
+      &:has(>sub)>sup {
+        letter-spacing: -0.05rem;
+      }
+    }
+    &:has(sub) {
+      &:after {
+        content: "\00a0";
```

Comment:
Can we just replace this in the actual method, rather than doing this with CSS?

See here: https://github.com/mealie-recipes/mealie/blob/b3241d3e8b7e708e0442d8d09d2c851fff765705/frontend/composables/recipes/use-recipe-ingredients.ts#L66C33-L66C37

Reply:
That makes a lot of sense indeed. After some testing, it turns out the extra space was not really needed at all, so I just removed it.